.storybook/index.scss

```
@@ -35,3 +37,26 @@
     }
   }
 }
+
+@include carbon--theme($carbon--theme--g10) {
+  .page-layouts__banner {
+    margin-top: 0;
+    max-width: 100%;
+    background-color: $inverse-02;
+    border-left: 3px solid $inverse-support-04;
+    color: $inverse-01;
+
+    svg.bx--inline-notification__icon {
+      fill: $inverse-support-04;
+    }
+
+    button.bx--btn--ghost {
+      color: $inverse-link;
+    }
+
+    button.bx--btn--ghost:hover,
+    button.bx--btn--ghost:focus {
+      background-color: $hover-secondary;
+    }
+  }
+}
```

We have had some challenges with inline theming in the past, so luckily we have our own helper mixin for this! I believe something like the following should work:

```suggestion
.page-layouts__banner {
  @include security--generate-css-custom-properties(
    $theme: $carbon--theme--g10
  );

  max-width: 100%;
    margin-top: 0;
}
```